<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigSQLConnection</class>
 <widget class="QDialog" name="ConfigSQLConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config SQL Connection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbSqlIp">
       <property name="text">
        <string>SQL IP Adresse</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="tbSqlIp"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbSqlPort">
       <property name="text">
        <string>SQL Port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="tbSqlPort"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbSqlUser">
       <property name="text">
        <string>SQL User</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="tbSqlUser"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbSqlPw">
       <property name="text">
        <string>SQL Password</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="tbSqlPw"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbSqlDb">
       <property name="text">
        <string>SQL Datenbank</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="tbSqlDb"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QPushButton" name="btSave">
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="btCheck">
       <property name="text">
        <string> Check Connection </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="btClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
